src/styles/stars.scss

```
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+  &--1 &__star:first-child,
```

```suggestion
  &--1 &__star:nth-child(-n + 1),
```

Make all code the same style